examples/files/readme.md

```diff
@@ -0,0 +1,87 @@
+## The ipfs files API
+
+Ipfs has an exciting new (ish) API called the 'files' API. It allows you to
+have a similar interface to traditional filesystems (mutable files and
+directories) while being backed by ipfs. In this post, I will give a few useful
+examples on what is possible.
+
+Lets start by making a couple directories and a few files.
+
```

Comment:
it would be good to explain that every node has a virtual `directory`, you can see it with `ipfs files ls /`